man/systemd.network.xml

```diff
@@ -1743,6 +1743,22 @@
           </listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term><varname>SendVendorOption=</varname></term>
+          <listitem>
+            <para>Send an arbitrary vendor option in the DHCPv4 request. Takes a DHCP option number, data type,
+            data separated and enterprise identifier with a colon
+            (<literal><replaceable>option</replaceable>:<replaceable>type</replaceable>:<replaceable>value</replaceable>
+            :<replaceable>enterprise identifier</replaceable></literal>).
```

Comment:
shouldn't the enterprise identifier come first? i.e. same order as in dhcp message?